src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs

```diff
@@ -1742,33 +1742,11 @@ private static int LocateLastFoundByte(Vector<byte> match)
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private static int LocateFirstFoundByte(ulong match)
-        {
-            if (Bmi1.X64.IsSupported)
-            {
-                return (int)(Bmi1.X64.TrailingZeroCount(match) >> 3);
-            }
-            else
-            {
-                // Flag least significant power of two bit
-                ulong powerOfTwoFlag = match ^ (match - 1);
-                // Shift all powers of two into the high byte and extract
-                return (int)((powerOfTwoFlag * XorPowerOfTwoToHighByte) >> 57);
```

Comment:
`BitOperations.TrailingZeroCount(ulong)` is now hardware accelerated on all x64 and ARM64.  On x86, the value is decomposed and accelerated as a 32-bit call, leaving only ARM32 as the only candidate for this software fallback.  Since it's doing 64-bit math, the decomposed LUT fallback in `BitOperations` is probably cheaper.